### PR TITLE
Issue 31 Make vlf-batch-size configurable in the remote case

### PR DESCRIPTION
**Pull request by danrharms**
Allow customization of the default batch size when accessing files remotely.

### CODE REVIEW

File: vlf-tune.el [hunk 1]
```diff
@@ -35,6 +35,11 @@
   :group 'vlf :type 'integer)
 (put 'vlf-batch-size 'permanent-local t)
 
+(defcustom vlf-batch-size-remote 1024
```
Comment (m00natic) [inline vlf-tune.el]: Better move this to vlf.el as it is used only there.

File: vlf-tune.el [hunk 2]
```diff
@@ -35,6 +35,11 @@
...
+  "Defines size (in bytes) of a batch of file data when accessed remotely."
+  :group 'vlf :type 'integer)
+(put 'vlf-batch-size-remote 'permanent-local t)
```
Comment (m00natic) [inline vlf-tune.el]: permanent-local is not needed in this case.  This customization is to
be used only once per VLF invocation, setting _initial_ value
vlf-batch-size, which has life on its own later.

File: vlf.el [hunk 3]
```diff
@@ -173,7 +173,7 @@ Return newly created buffer."
     (set-visited-file-name file)
     (set-buffer-modified-p nil)
     (if (or minimal (file-remote-p file))
-        (set (make-local-variable 'vlf-batch-size) 1024))
+        (set (make-local-variable 'vlf-batch-size) vlf-batch-size-remote))
```
Comment (m00natic) [inline vlf.el]: Split the minimal and remote-file-p cases.  Minimal should stay 1024.
It's meant for internal purposes (like batch operations).

**Comment by m00natic** [comment]
Perfect!

Only, can you squash the 2 commit as one ;-)

**Comment by danrharms** [comment]
I rebased and force pushed. Let me know if I didn't do this correctly. 

Out of curiosity, isn't there an option as of recently of doing a squash merge when merging in the pull request?  I haven't done many pull request revisions, so I'm curious about the different workflows being used.

**Comment by m00natic** [comment]
Ah, my bad, there indeed is such option (rebase squash).  I'm also
rarely dealing with pull requests and am blissfully ignorant of these
new fangled UI facilities.  And being cautious to press a button if not
sure what would happen exactly.

**Pull request merged**